guides/quickstart.md

```diff
@@ -0,0 +1,175 @@
+# Quickstart: How to configure and run the analyzer
```

Comment:
I think this is also a good opportunity to introduce the levee analogy. WDYT?

Reply:
Maybe. :thinking: I want this doc to be as concise as possible, and I'm not sure how to work that in.

It's a bit subtle, but the analogy is introduced in the `README`, at the top:

```
This static analysis tool works to ensure your program's data flow does not spill beyond its banks.
```

Do you have a suggestion?

Reply:
Something like 
> before we get into the working of levee we want to explain why we decided to call it so. If you think of you programs data as water that flows from one place to another then like a levee ensures that the water flow does not spill beyond its banks, go-flow-levee ensures that your program's data flow does not spill beyond its banks.

Reply:
Thank you. I'm not sure how that fits into "how to configure and run the analyzer", though. The intent for this guide is to show users how to get up and running quickly, so I don't think users will be interested in reading about the origin of the name of the tool.

@PurelyApplied, @benhxy, thoughts?

Reply:
I feel like we would expect a user to arrive here from the root `README.md`, which already contains that analogy in the header.  I'm not sure repeating it here is necessary.  \</2¢\>